js/digifinex.js

```diff
@@ -395,29 +474,43 @@ module.exports = class digifinex extends Exchange {
             // const status = this.safeString (market, 'status');
             // const active = (status === 'TRADING');
             //
-            const isAllowed = this.safeInteger (market, 'is_allow', 1);
-            const type = (defaultType === 'margin') ? 'margin' : 'spot';
-            const spot = (defaultType === 'spot') ? true : undefined;
-            const margin = (defaultType === 'margin') ? true : undefined;
+            let isAllowed = this.safeInteger (market, 'is_allow', 1);
+            let type = (defaultType === 'margin') ? 'margin' : 'spot';
+            const spot = settle === undefined;
+            const swap = !spot;
+            const margin = (marginMode !== undefined) ? true : undefined;
+            let symbol = base + '/' + quote;
+            let isInverse = undefined;
+            let isLinear = undefined;
+            if (swap) {
```

Comment:
@Dan-krm I think we have here some fields that could be used for swap markets 
![image](https://user-images.githubusercontent.com/43336371/199703225-c827ddc5-3936-43d7-a17e-45bbec1a1467.png)

Reply:
price_precision is used already, I’m thinking maybe leverage max should be saved for fetchLeverageTiers since it returns objects in the format:
[
  { leverage: '3', max_limit: '1000000' },
  { leverage: '10', max_limit: '500000' },
  { leverage: '50', max_limit: '200000' }
]
Unless you want me to iterate through and pick the largest leverage value for each given contract.

I added contract_value as the contractSize, and I added tick_size as the precision[‘amount’] but the tick_size is returning null with my output when I used parseNumber and parsePrecision with it and I’m not sure yet what the issue is there

Also I was thinking maybe I should add tick_size to limits['price']['min'] or limits['cost']['min']

Let me know what you think and if I should change any of these fields

Reply:
@Dan-krm,

> Unless you want me to iterate through and pick the largest and smallest leverage value for each given contract.

yes would be good if you could do this, if the list is sorted you just need to get the first and the last element


> Also I was thinking maybe I should add tick_size to limits['price']['min'] or limits['cost']['min']

I think it should be added to the price->min

Reply:
I did some testing and looking at the digifinex interface and those leverage values are actually just the leverage tiers and they're incomplete, for example it says 50 is the min and max leverage for BTC/USDT:USDT but leverage values of 1 to 100 are allowed

Reply:
@Dan-krm ok in that case let's leave it undefined here